tests/snos.rs

```diff
@@ -54,6 +57,7 @@ fn custom_hint_ok() {
         &mut hint_processor,
     )
     .expect("Couldn't run program");
+    compare_python_output(program_path, virtual_machine);
 }
```

Comment:
let's add a test for a case where it fails too, we need better error handling or it's gonna be a mess later on